### ISSUE TITLE
External For Profit Resources

### ISSUE THREAD

**Issue by quasiben** [dask/community#14]
In https://github.com/dask/dask/issues/5017 we are discussing building a list of educational resources for dask.  The topic of whether to include a for profit learning company came up and I wanted to isolate the topic in this issue by asking, Should the dask documentation/org link to external for profit resources ?  This could be beneficial for those interested in education, consulting, and products  around dask. However, this can also be read as an endorsement of those services and products.  It may also require some effort on the maintainers evaluating the products before we post a link.

I vaguely recall the Python.org site linking to other consulting/education services but I can no longer find the link -- maybe they linked to https://www.pythonanywhere.com for a quickstart ?

Thoughts ?

**Comment by mrocklin** [comment]
I'm generally in favor of engaging for-profit institutions where they benefit the project and its users.  Many of Dask's users are institutions, and generally I think that institutions are well-served by for-profit companies offering training courses, consulting, and paid products.  We have already provided good words around companies like Quansight, Anaconda, and Prefect.

However, in that issue the company in question is Datacamp, which might be a special case.  My concern about them wasn't that they were for-profit, it's rather that there was a bit of a scandal with them earlier this year (doing a web search for "datacamp ceo" will provide details).  Many of their instructors have publicly encouraged people to stop using Datacamp and there has been a general outcry against the company.  So in a case like Datacamp in particular I think that this would need more scrutiny.  To be honest I haven't kept up with the situation after the first couple of months.  It may be that the company has taken more significant steps and is now acting well.  If so I would be in favor of advertising their Dask course (it's good).  I would want to see more investigation by someone here before we recommended them to users though.
